source/js/diva.js

```diff
@@ -153,14 +153,22 @@ class Diva
         }
         else
         {
+        	let requestHeaders =  {
+                    "Accept": this.settings.acceptHeader,
+            }
+        	if (this.settings.addRequestHeaders) 
```

Comment:
Do we need `settings.addRequestHeaders` and `settings.acceptHeader`? Could you condense these to `settings.requestHeaders` in the settings, and then add whatever request headers you want when the viewer is initialized?

Reply:
Also, if you're adding a setting please add it to the list of settings above

Reply:
`settings.acceptHeader` is an existing setting that is just the string value of the accept header. I don't know if people use it as it is. 

I can replace it with a `settings.requestHeaders` that takes multiple headers as an object.